src/test/suite/utils/helper.ts

```diff
@@ -14,10 +14,12 @@ export type BinaryGenericRequest<T> = {
  * Activates the vscode.lsp-sample extension
  */
 export async function activate(
-  docUri: vscode.Uri
+  docUri?: vscode.Uri
 ): Promise<{ editor: TextEditor; doc: TextDocument } | null> {
   try {
-    const doc = await vscode.workspace.openTextDocument(docUri);
+    const doc = docUri
+      ? await vscode.workspace.openTextDocument(docUri)
+      : await vscode.workspace.openTextDocument(); // opens an untitled document
```

Comment:
I don't think you need to do that, if `docUri` is `undefined`, calling `vscode.workspace.openTextDocument(docUri)` is equivalent to calling it without arguments.

Reply:
if `docUri` is undefined, it'd be considered the same as calling the function without any arguments --> the overload will resolve to the first one.

give it a try, i'm pretty sure it'll work.

Reply:
oh well, then you can resolve this comment.
sorry for bothering you 🙈

Reply:
this is why I hate TS :(